src/main/java/com/google/sps/servlets/LoadInterviewsServlet.java

```diff
@@ -0,0 +1,256 @@
+
+    for (PossibleInterviewSlot slot : possibleInterviews) {
+      for (ScheduledInterview userInterview : userScheduledInterviews) {
+        if (userInterview.when().contains(Instant.parse(slot.utcEncoding()))) {
+          conflictingInterviews.add(slot);
+        }
+      }
+    }
+
+    possibleInterviews.removeAll(conflictingInterviews);
+
+    List<PossibleInterviewSlot> possibleInterviewList =
+        new ArrayList<PossibleInterviewSlot>(possibleInterviews);
+
+    sortInterviews(possibleInterviewList);
+    return possibleInterviewList;
+  }
+
+  private List<PossibleInterviewSlot> getPossibleInterviewSlotsForPerson(
+      String userId, TimeRange range, ZoneOffset timezoneOffset) {
+    List<Availability> availabilities =
+        availabilityDao.getInRangeForUser(userId, range.start(), range.end());
+    availabilities.removeIf(avail -> avail.scheduled());
+
+    List<PossibleInterviewSlot> possibleInterviewSlotsForPerson =
+        new ArrayList<PossibleInterviewSlot>();
+    for (int i = 0; i < availabilities.size() - 3; i++) {
+      if (availabilities
+          .get(i)
+          .when()
+          .start()
+          .plus(45, ChronoUnit.MINUTES)
+          .equals(availabilities.get(i + 3).when().start())) {
+        possibleInterviewSlotsForPerson.add(
+            PossibleInterviewSlot.create(
+                availabilities.get(i).when().start().toString(),
+                getDate(availabilities.get(i).when().start(), timezoneOffset),
+                getTime(availabilities.get(i).when().start(), timezoneOffset)));
+      }
+    }
+    return possibleInterviewSlotsForPerson;
+  }
+
+  private String getDate(Instant instant, ZoneOffset timezoneOffset) {
+    ZonedDateTime day = instant.atZone(ZoneId.ofOffset("UTC", timezoneOffset));
+    String dayOfWeek = day.getDayOfWeek().getDisplayName(TextStyle.FULL, Locale.US);
+    int month = day.getMonthValue();
+    int dayOfMonth = day.getDayOfMonth();
+    return String.format("%s %d/%d", dayOfWeek, month, dayOfMonth);
+  }
+
+  private String getTime(Instant instant, ZoneOffset timezoneOffset) {
+    ZonedDateTime startTime = instant.atZone(ZoneId.ofOffset("UTC", timezoneOffset));
+    ZonedDateTime endTime = startTime.plus(1, ChronoUnit.HOURS);
+    return String.format("%s - %s", formatTime(startTime), formatTime(endTime));
+  }
+
+  private String formatTime(ZonedDateTime time) {
+    int hour = time.getHour();
+    int minute = time.getMinute();
+    int standardHour = hour;
+    if (hour > 12) {
+      standardHour = hour - 12;
+    }
+    return String.format("%d:%02d %s", standardHour, minute, hour < 12 ? "AM" : "PM");
+  }
+
+  private void sortInterviews(List<PossibleInterviewSlot> possibleInterviewSlots) {
+    possibleInterviewSlots.sort(
+        (PossibleInterviewSlot p1, PossibleInterviewSlot p2) -> {
+          if (Instant.parse(p1.utcEncoding()).equals(Instant.parse(p2.utcEncoding()))) {
```

Comment:
Best not to parse these values a bunch of times. 

Declare them in advance and then reference them.